backend/Helpers/Validator.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Text.RegularExpressions;
+using System.Net.Mail;
+
+namespace OpenData.Helpers
+{
+    public static class Validator
```

Comment:
This also should be either dependency injected, or it should be a function on user that throws an exception if you try to create it with an invalid email

Reply:
Resolved in newer commit.